lib/redis/connection/ruby.rb

```diff
@@ -2,6 +2,7 @@
 require "redis/connection/command_helper"
 require "redis/errors"
 require "socket"
+require "openssl"
```

Comment:
Speaking from direct experience with Bundler, _tons_ of people have Ruby installed without being able to require `openssl`. 😕 Since Ruby will happily compile and install without any OpenSSL support, I suspect you'll want to `rescue LoadError` here.